Sources/Commands/SwiftTool.swift

```diff
@@ -431,7 +431,11 @@ public class SwiftTool<Options: ToolOptions> {
 
     private func printDiagnostics() {
         for diagnostic in diagnostics.diagnostics {
-            print(diagnostic: diagnostic)
+            if diagnostic.localizedDescription.hasPrefix("noManifest") {
```

Comment:
Ah, instead of doing this, we need to conform the enum that has this case with `CustomStringConvertible` protocol.

Reply:
actually i wont able to find noManifestCase in Diagnostic

Reply:
ideally _handle() should catch error for this case but diagnostics is catching this

Reply:
should i make a new case for noManifest in Diganostic

Reply:
noManifest is defined here: https://github.com/apple/swift-package-manager/blob/master/Sources/PackageModel/Package.swift#L90

We should also add a test case that demonstrates the fix!

Reply:
yeah and this should handle case for noManifest https://github.com/apple/swift-package-manager/blob/master/Sources/Commands/Error.swift#L66 But https://github.com/apple/swift-package-manager/blob/b78f787ff7c407d89fe41822fd6af7c23d1c4764/Sources/Commands/SwiftTool.swift#L419 is catching this

Reply:
i am having hard time to finding why it is not throwing a PackageModel.Package.Error.noManifest

Reply:
Yeah, that is some of the legacy code. I suggest you move [this](https://github.com/apple/swift-package-manager/blob/master/Sources/Commands/Error.swift#L67-L70) to https://github.com/apple/swift-package-manager/blob/master/Sources/PackageModel/Package.swift#L89